### PR TITLE
[search-tests-support][indexer-tests-support] TestWithCustomMwms.

### PR DESCRIPTION
**Pull request by ygorshenin**

### CODE REVIEW

File: indexer/indexer_tests_support/test_with_custom_mwms.hpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  static void Cleanup(platform::LocalCountryFile const & file);
+
+  virtual void OnMwmBuilt(MwmInfo const & /* info */) {}
+
+  Index m_index;
+  std::vector<platform::LocalCountryFile> m_files;
+};
+}  // namespace tests_support
+}  // namespace indexer
+
```
Comment (mpimenov) [inline indexer/indexer_tests_support/test_with_custom_mwms.hpp]: One newline is enough.

File: indexer/indexer_tests_support/test_with_custom_mwms.hpp [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+#include "platform/country_file.hpp"
+#include "platform/local_country_file.hpp"
+#include "platform/platform.hpp"
+
+#include "base/assert.hpp"
+
+#include <string>
+#include <vector>
+#include <utility>
```
Comment (mpimenov) [inline indexer/indexer_tests_support/test_with_custom_mwms.hpp]: sort, please

File: search/search_tests_support/test_with_custom_mwms.hpp [hunk 3]
```diff
@@ -1,93 +1,22 @@
-#pragma once
```
Comment (mpimenov) [inline search/search_tests_support/test_with_custom_mwms.hpp]: ?

File: indexer/indexer_tests_support/test_with_custom_mwms.hpp (hunk 1 above)
Comment (ygorshenin) [reply indexer/indexer_tests_support/test_with_custom_mwms.hpp]: Done

File: indexer/indexer_tests_support/test_with_custom_mwms.hpp (hunk 2 above)
Comment (ygorshenin) [reply indexer/indexer_tests_support/test_with_custom_mwms.hpp]: Done

File: search/search_tests_support/test_with_custom_mwms.hpp (hunk 3 above)
Comment (ygorshenin) [reply search/search_tests_support/test_with_custom_mwms.hpp]: Done

**Pull request merged**